core/src/main/java/com/afollestad/materialdialogs/utils/MDUtil.kt

```diff
@@ -234,6 +234,13 @@ object MDUtil {
     }
   }
 
+  @RestrictTo(LIBRARY_GROUP) fun TextView?.maybeSetAdditionalPadding(height: Int): Int {
```

Comment:
The receiver `TextView` doesn't need to be nullable here, since it's only used in `DialogTitleLayout` and the text view there is never null 😁